### DISCUSSION TITLE
İBAX cüzdanımızda farklı coinler saklayabilme ve stake özelliği

### DISCUSSION THREAD

**Discussion by mamilincoln** [IBAX-io/go-ibax#1854] [Ideas]
### Clear and concise description of the problem

İBAX cüzdanımıza farklı coinler transfer edip saklayabilme yada stake edebilme özelliği olabilir mi acaba?

### Suggested solution

İBAX cüzdanımıza farklı coinler transfer edip saklayabilme yada stake edebilme özelliği olabilir mi acaba?

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by metehanguler34** [comment]
Sanmıyorum

**Comment by FurkanOzay** [comment]
Doğru anladıysam kendi ağlarında yapacakları proje tokenlarını tutabiliriz sanırım!

**Comment by Grkmkgl** [comment]
Idax bi saklayalimda gerisine sonra bakariz
